microprofile/WFLY-15016-mp-reactive-messaging-amq-streams-support.adoc

```diff
@@ -0,0 +1,88 @@
+= Full Support for MicroProfile Reactive Messaging for AMQ Streams (Kafka) integration
+:author:            Kabir Khan
+:email:             kkhan@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+AMQ Streams is Red Hat's offering of Kafka on OpenShift. Users use the AMQ Streams operator to create a Kafka instance in their OpenShift setup.
+
+Most of the work has been done in the related issues below, so this RFE is more about testing and documenting.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-15016[WFLY-15016]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-14932[WFLY-14932 - Upgrade to MicroProfile Reactive Messaging 2.0] - This version introduces the new `@Channel` annotation, which makes it easier for users to get data into/from the Reactive Messaging Streams
```

Comment:
Wrong issue here, it should be the following one: https://issues.redhat.com/browse/WFLY-14798